cloud_env.go

```diff
@@ -0,0 +1,325 @@
+/*
+ * © Copyright IBM Corp. 2018
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package IBMCloudEnv
+
+import ("os"
+		"fmt"
```

Comment:
fix indentation in this import

Reply:
It looks like this whole file needs to be formatted so indentations are consistent and correct.